src/static_julia.jl

```diff
@@ -156,36 +162,43 @@ function julia_flags(optimize, debug, cc_flags)
 end
 
 function build_julia_cmd(
-        sysimage, compile, cpu_target, optimize, debug, inline, check_bounds,
-        math_mode, depwarn, startupfile = false
+        sysimage, precompiled, compilecache, home, startup_file, handle_signals,
+        compile, cpu_target, optimize, debug, inline, check_bounds, math_mode, depwarn
     )
+    # TODO: these may just be temporary workarounds, see: https://github.com/JuliaLang/PackageCompiler.jl/issues/47
+    precompiled == nothing && cpu_target == nothing || (precompiled = "no")
+    compilecache == nothing && (compilecache = "no")
```

Comment:
hmm, i could be misunderstanding, but the first one doesn't feel correct to me. 
If the user passes `precompiled = "yes"`, won't this set precompiled to "no"?

I think maybe you want parentheses around the second two phrases?
```
 precompiled == nothing && (cpu_target == nothing || (precompiled = "no"))
```
But honestly that's still hard to parse. perhaps this can simply be an `if`-statement? I think this is what you meant?:
```
if (precompiled == nothing && cpu_target != nothing) precompiled = "no" end
# or this:
(precompiled == nothing && cpu_target != nothing) && (precompiled = "no")
```